src/Opserver.Web/Opserver.Web.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>Opserver</RootNamespace>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\Opserver.Core\Opserver.Core.csproj" />
+    <PackageReference Include="BuildBundlerMinifier" Version="3.2.435" PrivateAssets="all" />
+    <PackageReference Include="BuildWebCompiler" Condition="'$(OS)' == 'Windows_NT'" Version="1.12.405" PrivateAssets="all" />
+    <PackageReference Include="MiniProfiler.AspNetCore.Mvc" Version="4.1.0" />
+    <PackageReference Include="StackExchange.Exceptional.AspNetCore" Version="2.1.0" />
+    <PackageReference Include="System.DirectoryServices.AccountManagement" Version="4.7.0" />
+    <Reference Include="System.Management" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="Config\*.config.example" />
```

Comment:
FYI This didn't include any files on my machine - i.e. it was an empty folder.

Reply:
Yep, intended - it's a glob for if there's anything there in Solution Explorer :)